nexus/src/nexus.rs

```diff
@@ -1973,7 +1979,11 @@ impl Nexus {
      */
 
     fn as_rack(&self) -> db::model::Rack {
-        db::model::Rack { identity: self.api_rack_identity.clone() }
+        db::model::Rack {
+            identity: self.api_rack_identity.clone(),
+            tuf_metadata_base_url: "http://localhost:8000/metadata".to_string(),
+            tuf_targets_base_url: "http://localhost:8000/targets".to_string(),
```

Comment:
These might also be good candidates for the Nexus config file?

Also, should we have instructions on "how to set this up"? I don't think these servers are running on most people's machines.

Reply:
I've just been running `python3 -m http.server` in the output directory from tuftool. (Instructions for making a repo probably should go along with that.)

These maybe should belong to the Nexus config file to start, but these eventually need to be configurable by an operator... thoughts?

Reply:
What's the longer term plan here?  Are we building a new component to serve this endpoint?  Is it part of the control plane?  If not, maybe we want to treat this like we do Clickhouse and CockroachDB, with easy ways to run it by hand that we use in the test suite to get coverage of this stuff?

Reply:
Longer-term plan is for this to default to something.oxide.computer via HTTPS and be stored in the database, ready for an end-user to change it if they need to.